src/components/battles/animations/shared/items/Item.ts

```diff
@@ -0,0 +1,86 @@
+import { IMoveAction, IMoveEffect, ITeamAndAction, Team } from "battlemovr";
+import { GeneralComponent } from "gamestartr";
+
+import { FullScreenPokemon } from "../../../../../FullScreenPokemon";
+import { IBattleInfo, IPokemon } from "../../../../Battles";
+import { IMenu } from "../../../../Menus";
+import { IThing } from "../../../../Things";
+
+/**
+ * Uses an item.
+ */
+export class Item<TGameStartr extends FullScreenPokemon> extends GeneralComponent<TGameStartr> {
```

Comment:
It's confusing to me how this is called `Item` but refers to the animation (`Move` is also confusing, but I can fix that later). Let's rename it to `ItemAnimation`.